apps/nextjs-website/src/app/solutions/[solutionSlug]/[...solutionSubPathSlugs]/page.tsx

```diff
@@ -64,6 +60,7 @@ const Page = async ({ params }: { params: Params }) => {
     params?.solutionSubPathSlugs
   );
 
+  const urlReplacesMap = (await getUrlReplaceMapProps()) || {};
```

Comment:
it seems it's declared but not used

Reply:
it's used later, line 79, to build the props ( i just changed the declaration with the new map)